lib/spectator/counter.rb

```diff
@@ -20,7 +20,12 @@ def increment(delta = 1)
 
     # Get the current count as a list of Measure and reset the count to 0
     def measure
-      [Measure.new(@id.with_stat('count'), @count.get_and_set(0))]
+      cnt = @count.get_and_set(0)
+      if cnt.positive?
+        [Measure.new(@id.with_default_stat('count'), cnt)]
+      else
+        []
```

Comment:
> To simplify this PR we now ensure that counters with no activity do not return an empty set of measurements.

It seems like before this change a counter would always have a measurement that could be 0. Now if there was no activity it would be empty list. Should that be "ensure that counters with no activity return an empty set of measurements"?